specs/AppLifecycle/EnvironmentVariables/AppLifecycle Environment Variables.md

```diff
@@ -10,12 +10,13 @@ Environment variables is one example, where there are several existing APIs
 that provide partially overlapping functionality, and with some gaps. This spec
 describes the Reunion support for environment variables.
 
-Important Note: this feature relies on other platform features which are only
-available from Windows summer 2021 onwards. Therefore this feature is also 
+Important Note: The EV tracking feature relies on other platform features which are only
```

Comment:
`EV` used but not defined

Always define an acronym before later usage

SUGGESTION: Change title to `AppLifecycle - Environment Variables (EV)`